docs/man/containerd-config.toml.5.md

```diff
@@ -209,6 +226,12 @@ imports = ["/etc/containerd/runtime_*.toml", "./debug.toml"]
   [plugins."io.containerd.service.v1.tasks-service"]
     blockio_config_file = ""
     rdt_config_file = ""
+  [plugins.'io.containerd.image-verifier.v1.bindir']
```

Comment:
Comparing 1.0 and 2.0 using vscode compare
<img width="1043" alt="Screenshot 2024-11-01 at 8 38 07 AM" src="https://github.com/user-attachments/assets/1a72af3f-80f5-4202-85f6-6e810b3c320f">